qemu/tests/virtio_serial_hotplug_port_pci.py

```diff
@@ -13,6 +13,21 @@
 from qemu.tests.vioser_in_use import live_migration_guest  # pylint: disable=W0611
 
 
+def get_buses_serial_devices(vm, params, char_devices, serials):
```

Comment:
Doc string, and recommend to add `and` between `buses` and `serial_devices`

Reply:
Thanks and update.